websites/R/Richup.io/presence.ts

```diff
@@ -0,0 +1,98 @@
+const presence = new Presence({
+		clientId: "1067910906912194590",
+	}),
+	pages: { [k: string]: string } = {
+		"/settings": "Settings",
+		"/store/player-appearance": "Player Appearance",
+		"/store/board-maps": "Board Maps",
+		"/store/upgrades": "Upgrades",
+		"/store/avatars": "Profile Pictures",
+		"/store/coins": "Coins",
+	};
+
+let startTimestamp: number;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "rio-logo",
```

Comment:
Imgur links are preferred over Discord asset keys now.
```suggestion
			largeImageKey: "https://i.imgur.com/4Nw3mle.png",
```

Reply:
weird but ok